instrumentation/redisson-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/redisson/RedissonRequest.java

```diff
@@ -5,17 +5,91 @@
 
 package io.opentelemetry.javaagent.instrumentation.redisson;
 
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
 import com.google.auto.value.AutoValue;
+import io.netty.buffer.ByteBuf;
+import io.opentelemetry.instrumentation.api.db.RedisCommandSanitizer;
 import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.redisson.client.protocol.CommandData;
+import org.redisson.client.protocol.CommandsData;
 
 @AutoValue
 public abstract class RedissonRequest {
 
+  public static RedissonRequest create(InetSocketAddress address, Object command) {
+    return new AutoValue_RedissonRequest(address, command);
+  }
+
   public abstract InetSocketAddress getAddress();
 
   public abstract Object getCommand();
 
-  public static RedissonRequest create(InetSocketAddress address, Object command) {
-    return new AutoValue_RedissonRequest(address, command);
+  public String getOperation() {
+    Object command = getCommand();
+    if (command instanceof CommandData) {
+      return ((CommandData<?, ?>) command).getCommand().getName();
+    } else if (command instanceof CommandsData) {
+      CommandsData commandsData = (CommandsData) command;
+      if (commandsData.getCommands().size() == 1) {
+        return commandsData.getCommands().get(0).getCommand().getName();
+      }
+    }
+    return null;
+  }
+
+  public String getStatement() {
+    List<String> sanitizedStatements = sanitizeStatement();
+    switch (sanitizedStatements.size()) {
+      case 0:
+        return null;
```

Comment:
this used to return the text "Redis Command", but since `db.statement` is not required, I think makes sense to return `null` and not set it instead